test/cpp/end2end/xds/xds_end2end_test_lib.h

```diff
@@ -164,6 +165,10 @@ class XdsTestType {
     return info.param.AsString();
   }
 
+  friend void PrintTo(const XdsTestType& test_type, std::ostream* os) {
```

Comment:
Why is this needed?  We should be using the `Name()` method whenever we call `INSTANTIATE_TEST_SUITE_P()`.

Reply:
It's for failing test:
No change:
```
[  FAILED  ] XdsTest/ClientStatusDiscoveryServiceTest.XdsConfigDumpMultiClient/V3BootstrapFromEnvVar, where GetParam() = 20-byte object <00-00 00-00 00-00 00-00 02-00 00-00 02-00 00-00 00-00 00-00>
```
With this change:
```
[  FAILED  ] XdsTest/ClientStatusDiscoveryServiceTest.XdsConfigDumpMultiClient/V3BootstrapFromEnvVar, where GetParam() = V3BootstrapFromEnvVar
```
I now see that filter can be setup, I did not notice there was a parameter value after the slash.

Reply:
The fact that there's a parameter after the slash is exactly why I don't think we need this.  The log line that you quoted shows that this string is already logged.

But I'll leave it up to you.